src/utils/Meta.js

```diff
@@ -28,9 +30,14 @@ const META = {
   isElement: ({_meta}) => _meta.type === META.type.element,
   isView: ({_meta}) => _meta.type === META.type.view,
   isModule: ({_meta}) => _meta.type === META.type.module,
+  isType: ({_meta}, type) => _meta.type === type,
 
   // parent
-  isChild: ({_meta}) => !!_meta.parent,
+  isParent: (component) => (
+    // has no 'parent' or 'parent' is self
+    !_.has(component, '_meta.parent') || _.get(component, '_meta.parent') === component._meta.name
+  ),
+  isChild: component => !META.isParent(component),
```

Comment:
Extended META to include helper method for identifying a parent component and a component with a type string.